cmd/net.go

```diff
@@ -77,18 +77,58 @@ func getHostIP4(host string) (ipList set.StringSet, err error) {
 	return ipList, err
 }
 
+// byLastOctetValue implements sort.Interface used in sorting a list
+// of ip address by their last octet value.
+type byLastOctetValue []net.IP
+
+func (n byLastOctetValue) Len() int      { return len(n) }
+func (n byLastOctetValue) Swap(i, j int) { n[i], n[j] = n[j], n[i] }
+func (n byLastOctetValue) Less(i, j int) bool {
+	return []byte(n[i].To4())[3] > []byte(n[j].To4())[3]
+}
```

Comment:
It is usually based on the last octet.  It is not meant to be a perfect replacement neither it is going to fully eradicate 127.0.0.1 to be printed as the first element,  it is for a best case scenario only. 

But your change is easy to incorporate let me do that.